src/app/_components/logo.tsx

```diff
@@ -0,0 +1,33 @@
+import Link from "next/link"
```

Comment:
`next/link` shouldn't be used for external links. if we plan on using external links, it should be in a separate component with the `target="_blank"` attribute. as a sidenote: external links should use `rel="noopener"` to prevent a `window.opener` vulnerability, but the default behavior in most modern browsers as of several years ago is to treat all external links with `noopener`. it may be good to include regardless, but the edge case would be so arcane (an out-of-date browser referring to a website that is compromised) it's not even worth mentioning, especially if it exclusively links to reputable websites